operator/src/main/java/oracle/kubernetes/operator/helpers/CallBuilder.java

```diff
@@ -630,7 +633,7 @@ private Call readDomainAsync(
    */
   public Step readDomainAsync(String name, String namespace, ResponseStep<Domain> responseStep) {
     return createRequestAsync(
-        responseStep, new RequestParams("readDomain", namespace, name, null), readDomain);
+        responseStep, new RequestParams("readDomain", namespace, name, null, name), readDomain);
```

Comment:
is domain name always the same as uid?

Reply:
It is the domain resource's metadata.getName(), which is the domainUID.

Reply:
Just clarifying this: .metadata.name is the default value for domainUID, but there is also a .spec.domainUid field that, if set, overrides .metadata.name.

The domain name from config.xml can be a different value.